src/GraphQL/Schemas/Primary/Substrate/Mutations/ApproveCollectionMutation.php

```diff
@@ -96,6 +99,7 @@ public function resolve(
                 'method' => $this->getMutationName(),
                 'encoded_data' => $encodedData,
                 'idempotency_key' => $args['idempotencyKey'] ?? Str::uuid()->toString(),
+                'simulate' => $args['simulate'] ?? false,
```

Comment:
Do we need a null coalesce if the arg has a defaultValue?  I believe it should always be set, even if not passed directly.
```suggestion
                'simulate' => $args['simulate'],
```

Reply:
Thanks, removed